scripts/run-ocdb-tests.sh

```diff
@@ -9,7 +9,7 @@ if [ "$(basename $PWD)" != "open-components-database" ] ; then
     exit -1
 fi
 
-PYTHON=python3.10
+PYTHON=python
```

Comment:
may I suggest `PYTHON=${PYTHON:-python}` instead for easy override?

Reply:
Sure, I did that in other scripts but missed this one.